content/en/contribute/code/core/automated-tests.md

```diff
@@ -115,3 +115,37 @@ Running e2e tests can be quite slow so to save time modify the `specs` property
 #### Watching the test run
 
 Running the tests locally with `npm run wdio-local` or `npm run standard-wdio-local` will allow you to watch it run but if you interact with the page the test will fail in unexpected ways. Furthermore the browser will close after a short timeout so you won't be able to inspect the console or DOM. To do this, force quit the process running the test before it tears down and you will be able to navigate around the app, use Chrome dev tools, and inspect the docs in the database to (hopefully) work out what's going wrong.
+
+#### Running the upgrade e2e test locally
+
+To run the upgrade e2e tests in your local environment, follow these steps:
+
+- Make sure your branch has been published, and it's available in the market:
+    - A way to do this is by pushing the branch, let the GitHubActions to run, if all the other e2e are okay, then it will publish the branch.
+    - Check that your branch name is available [here](https://staging.dev.medicmobile.org/_couch/builds_4/_design/builds/_view/releases).
+- Make sure to stop all existing containers
+- Set these environment variables:
+    - `export MARKET_URL_READ=https://staging.dev.medicmobile.org`.
+    - `export STAGING_SERVER=<builds path>`.
+      - To upgrade from CHT v4.x.x use `export STAGING_SERVER=_couch/builds_4`.
+      - To upgrade from previous versions of CHT use: `export STAGING_SERVER=_couch/builds`.
```

Comment:
I don't believe you can upgrade from previous versions, because those do not have docker compose files. We should remove this line.